lib/internal/util/inspect.js

```diff
@@ -650,18 +659,26 @@ function formatRaw(ctx, value, recurseTimes, typedArray) {
         return ctx.stylize(base, 'date');
       }
     } else if (isError(value)) {
-      // Make error with message first say the error.
-      base = formatError(value);
+      // https://tc39.github.io/ecma262/#sec-error.prototype.tostring
+      const name = value.name === undefined ? 'Error' : `${value.name}`;
+      const message = value.message === undefined ? '' :
+        `${strEscape(value.message, false)}`;
+      if (!name) {
+        base = message;
+      } else if (!message) {
+        base = name;
+      } else {
+        base = `${name}: ${message}`;
+      }
+      if (value.stack && ctx.indentationLvl === 0) {
```

Comment:
Would flipping the condition avoid triggering the `value.stack` creation on access  when `indentationLvl` is not `0`?
```js
ctx.indentationLvl === 0 && value.stack
```